src/hooks/util/parseMarkdown.tsx

```
@@ -0,0 +1,55 @@
+        return `<table class="border-collapse w-full my-3 text-sm"><thead><tr>${headers}</tr></thead><tbody>${bodyRows}</tbody></table>`;
+      })
+      // 줄바꿈
+      .replace(/\n(?!<)/g, '<br/>')
+  );
+}
+
+export default function MarkdownRenderer({ content }: { content: string }) {
+  return (
+    <div
+      className="text-sm leading-relaxed"
+      dangerouslySetInnerHTML={{ __html: parseMarkdown(content) }}
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)